### ISSUE TITLE
ChromeSocketXHR chunked response incorrect

### ISSUE THREAD

**Issue by jonaspu** [kzahel/web-server-chrome#118]
**Steps to reproduce**
Load any content that is sent using _Transfer-Encoding: Chunked_

**Bug**
The response contains a newline character after every chunk

**Example**
Loading _https://www.google-analytics.com/ga.js_ produces the following difference:
[Left: Original loaded with Chrome / Right: Loaded with ChromeSocketXHR](https://www.diffchecker.com/lao4qvmK)

**Comment by kzahel** [comment]
Thanks for catching this. You're right it looks like I'm not stripping out the extra newlines. I may strip them before I add them to the chunks buffer. I'll keep this issue updated when I get time to put in a fix!

**Comment by jonaspu** [comment]
Yes, this might be a better approach than just copy pasting a function an changing some minor stuff (like i did)

**Comment by kzahel** [comment]
The original Chrome App is no longer supported. The app has been rebuilt from scratch — see https://ok200.app for the new versions and to sign up for updates.
